tasks/update_log4j.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Replace DailyRollingFileAppender with RollingFileAppender
+  replace:
+    path: "{{log4j_file}}"
```

Comment:
How are permissions on the logging directories handled should the user be changed and or the logging directory customized?